dotnet/src/AutoGen.SemanticKernel/Extension/KernelExtension.cs

```diff
@@ -1,14 +1,14 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // KernelExtension.cs
 
-using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Agents;
 
 namespace AutoGen.SemanticKernel.Extension;
 
 public static class KernelExtension
 {
-    public static SemanticKernelAgent ToSemanticKernelAgent(this Kernel kernel, string name, string systemMessage = "You are a helpful AI assistant", PromptExecutionSettings? settings = null)
+    public static SemanticKernelAgent ToSemanticKernelAgent(this ChatCompletionAgent agent, string name, string systemMessage = "You are a helpful AI assistant")
     {
-        return new SemanticKernelAgent(kernel, name, systemMessage, settings);
+        return new SemanticKernelAgent(agent, name, systemMessage);
```

Comment:
Same here, please keep the old API to avoid breaking change. You can mark the old api with `Obsolete` so it will trigger a warning message on user end